### PR TITLE
Changed name of Size to Vector.

### PR DESCRIPTION
**Pull request by simonmika**
Renamed `IntSize2D`, `FloatSize2D`, `IntSize3D`, `FloatSize3D` to `IntVector2D`, `FloatVector2D`, `IntVector3D`, `FloatVector3D`,  in accordance with #783.

### CODE REVIEW

**Comment by simonmika** [comment]
@sebastianbaginski Can you peer review this?

File: source/draw/FloatImage.ooc [hunk 1]
```diff
@@ -3,13 +3,13 @@ use ooc-math
 FloatImage : class {
 	// x = column
 	// y = row
-	_size: IntSize2D
+	_size: IntVector2D
 	size ::= _size
 	_pointer: Float*
 	pointer ::= this _pointer
-	init: func ~IntSize2D (=_size)
+	init: func ~IntVector2D (=_size)
```
Comment (sebastianbaginski) [inline source/draw/FloatImage.ooc]: Is the suffix needed here ?

Comment (simonmika) [inline source/draw/FloatImage.ooc]: You are right that one of them _could_ drop the suffix. It seams to be the norm in the Image classes to have suffixes on all initiators.

Comment (sebastianbaginski) [inline source/draw/FloatImage.ooc]: OK, don't change it, I'm just asking

**Comment by sebastianbaginski** [comment]
I think it looks good. Do you want me to test it ?

**Comment by simonmika** [comment]
That would be great.

**Comment by sebastianbaginski** [comment]
`test.sh` ok. Review done.

**Pull request merged**